src/Sulu/Bundle/MediaBundle/Resources/config/storages/google_cloud.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <services>
+        <service id="sulu_media.storage.google_cloud.client" class="Google\Cloud\Storage\StorageClient">
+            <argument type="collection">
+                <argument key="keyFilePath">%sulu_media.media.storage.google_cloud.key_file_path%</argument>
+            </argument>
+        </service>
+
+        <service id="sulu_media.storage.google_cloud.bucket" class="Google\Cloud\Storage\Bucket">
```

Comment:
Doesn't that mean that we also have to add the google package as well to `composer.json` somehow?

Reply:
will add it to require-dev